cve_bin_tool/OutputEngine.py

```diff
@@ -62,6 +64,101 @@ def output_csv(self, outfile):
                     row = [modulename, version, cve_number, cve_severity]
                     writer.writerow(row)
 
+    def write_console(self, modulename, version, cve_number, cve_severity, outfile):
+        """ Output Module, Version, CVE_Number, Severity to the console in tabular form"""
+
+        # Max space for each tabular cell
+        MODULE_NAME_MAX = 18
+        VERSION_MAX = 9
+        CVE_NUMBER_MAX = 18
+        CVE_SEVERITY_MAX = 9
+
+        # Calculate length of -- modulename, version, cve_number, cve_severity
+        MODULE_NAME_LEN = len(modulename)
```

Comment:
```suggestion
        MODULE_NAME_LEN = len(module_name)
```

For consistency, if we're doing MODULE_NAME, then we should do module_name

Reply:
I think I must make as ```MODULE_NAME_LEN``` to ```MODULENAME``` because everywhere its modulename.